lib/util/npm-util.js

```diff
@@ -53,7 +53,7 @@ function installSyncSaveDev(packages) {
     if (Array.isArray(packages)) {
         packages = packages.join(" ");
     }
-    shell.exec(`npm i --save-dev ${packages}`, { stdio: "inherit" });
+    childProcess.execSync(`npm i --save-dev ${packages}`, { stdio: "inherit", encoding: "utf8" });
```

Comment:
Have you tested this on windows? I always have trouble using `childProcess.exec()` on windows with node packages or batch file.